supabase/functions/admin-update-user/index.ts

```diff
@@ -0,0 +1,75 @@
+// @ts-nocheck (for Deno references, annoyingly cannot get around for now)
+/* global Deno */
+
+import { type Handler, serve } from "https://deno.land/std@0.168.0/http/server.ts";
+import { createClient } from "https://esm.sh/@supabase/supabase-js@latest";
+import { corsHeaders } from "../_shared/cors.ts";
```

Comment:
@ import

Reply:
Can't @ import in edge functions

Reply:
Can't use `@` imports in edge functions, sadly